iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -114,6 +114,11 @@ iXBRLViewer.prototype.isReviewModeEnabled = function () {
     return this.isFeatureEnabled('review');
 }
 
+iXBRLViewer.prototype.isViewerEnabled = function () {
+    const urlParams = new URLSearchParams(window.location.search);
+    return (urlParams.get('disable-viewer') ?? 'false') === 'false'
```

Comment:
Should this optional functionality be included in the README? 

Similarly, I was wondering if we should add anything to the README for available command line options. Like, `--use-stub-viewer`?

Reply:
Docs for this added in [a492818](https://github.com/Arelle/ixbrl-viewer/pull/579/commits/a49281822746ff8899906e140fa46ba360d9c554)

README improvements are in #580